lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -207,11 +210,12 @@ protected function _getPortableTableIndexesList($tableIndexes, $tableName = null
             $idx['primary']    = false;
             $idx['non_unique'] = ! $tableIndex['unique'];
 
-                $stmt       = $this->_conn->executeQuery(sprintf(
-                    'PRAGMA INDEX_INFO (%s)',
-                    $this->_conn->quote($keyName)
-                ));
-                $indexArray = $stmt->fetchAll(FetchMode::ASSOCIATIVE);
+            $stmt       = $this->_conn->executeQuery(sprintf(
```

Comment:
```suggestion
            $stmt = $this->_conn->executeQuery(sprintf(
```

Because of the newline above this, I assume that the spaces aren't needed here.